onnxruntime/core/providers/cpu/nn/pool_base.h

```diff
@@ -106,7 +106,8 @@ class PoolBase {
 
  protected:
   PoolBase(const OpKernelInfo& info)
-      : op_name_(info.GetKernelDef().OpName()),
+      : op_name_(info.GetKernelDef().OpName().rfind("QLinear", 0) != 0 ?
```

Comment:
Will it have impact on other ops, like QLinearGlobalAveragePool? Is it better to handle it in the PoolAttributes

Reply:
Those are not affected. Also QLinear* is not using it as it is not supported here before.